### ISSUE TITLE
Missing status logfile prevents startup

### ISSUE THREAD

**Issue by sypherspace** [fuzzball-muck/fuzzball#66]
Attempting to start the muck while configured with `MUD_ID`/`MUD_GID` will result in failed start with the following message, if said file does not already exist:

`Unable to open logs/status!`

(Expected behavior would be to create the file.)

**Comment by sypherspace** [comment]
Correction: Files get created prior to the process dropping root privileges, which makes them inaccessible to the unprivileged process.

```
# ls -l
total 24
-rw------- 1 root     root        0 Feb  7 21:12 fbmuck
-rw------- 1 root     root       28 Feb  7 21:12 fbmuck.err
drwx------ 2 fuzzball nogroup 12288 Jan 27 06:25 old
-rw------- 1 root     root       22 Feb  7 21:12 restarts
-rw------- 1 root     root      119 Feb  7 21:12 status
```

**Comment by wyld-sw** [comment]
Merged issue with #56.
